dotcom-rendering/src/components/FootballTablesPage.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import type { Meta, StoryObj } from '@storybook/react/*';
+import { fn } from '@storybook/test';
+import { regions } from '../../fixtures/manual/footballData';
+import { Default as TableDefault } from './FootballTable.stories';
+import { FootballTablesPage } from './FootballTablesPage';
+
+const meta = {
+	title: 'Components/Football Tables Page',
+	component: FootballTablesPage,
+} satisfies Meta<typeof FootballTablesPage>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default = {
```

Comment:
Given this only has one story, you can give it the same name as the component so that it gets hoisted[^1] in the UI:

```suggestion
import { FootballTablesPage as FootballTablesPageComponent } from './FootballTablesPage';

const meta = {
	title: 'Components/Football Tables Page',
	component: FootballTablesPageComponent,
} satisfies Meta<typeof FootballTablesPageComponent >;

export default meta;

type Story = StoryObj<typeof meta>;

export const FootballTablesPage = {
```

[^1]: https://storybook.js.org/docs/writing-stories/naming-components-and-hierarchy#single-story-hoisting